diff --git a/dist/index.js b/dist/index.js
index 88dcd49d80acb70039477e8794d0e8fa7d0a3e89..8b58d62c178ad77348d7b7538c07bd134f87cd2d 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -727,7 +727,7 @@ function syncControls(component) {
   setArgObserver(component);
   setTimeout(() => {
     const selectedComponent = document.querySelector(component.tagName);
-    if (selectedComponent && argsObserver) {
+    if (selectedComponent && argObserver) {
       argObserver.observe(selectedComponent, {
         attributes: true
       });
